### ISSUE TITLE
guild.roles.forEach is not a function

### ISSUE THREAD

**Issue by dr3adx** [Androz2091/discord-backup#19]
Trying to create backup i get following error:

```
 (node:21278) UnhandledPromiseRejectionWarning: TypeError: guild.roles.forEach is not a function
    at Object.<anonymous> (/root/Desktop/discordjs/node_modules/discord-backup/lib/master/create.js:78:25)
```

Im using discordjs v12, bot has admin permissions and even admin role in server

seems like they changed some stuff in discordjs's API

**Comment by mystere19052003** [comment]
Hello, change guild.roles.forEach by guild.roles.cache.forEach
Normally it will work.

**Comment by Androz2091** [comment]
Thanks for reporting the problem, it should be fixed in `2.1.2`.
